docs/metadata-format.md

```diff
@@ -1,379 +1,282 @@
-<h1>Metadata format</h1>
+<<h1>Metadata format</h1>
 
-In your [data repository](glossary.md#data-repository) the metadata is maintained on an indicator-by-indicator basis. This metadata can include any number of custom fields, as defined in a [schema file](https://github.com/open-sdg/open-sdg-data-starter/blob/develop/_prose.yml) (see the "Schema" section below) in your data repository. Some fields, however, are mandatory and/or have specific uses in Open SDG. This page details those fields.
+In your [data repository](glossary.md#data-repository) the metadata is maintained on an indicator-by-indicator basis. 
+
+By default the metadata fields are based on SDG SDMX detailed metadata concepts and the template metadata file in the data starter is based on these fields. It is recommended that you use the default fields as they follow the format that is requested by the UN and allow metadata to compared. However, the metadata can include any number of custom fields, as defined in a [schema file](https://github.com/open-sdg/open-sdg-data-starter/blob/develop/metadata_schema.yml) (see the "Schema" section below) in your data repository. However, 
+
+Whilst it is possible to use the SDG SDMX main metadata concepts (which are a level above the detailed concepts) e.g. **0. Indicator information**, the UNSD strongly recommends that the detailed concepts are used.
```

Comment:
Maybe give an example of a detail concept, like you did for the `**0. Indicator information**` main concept?